drv/stm32h7-sprot-server/src/main.rs

```diff
@@ -438,7 +459,7 @@ impl<S: SpiServer> Io<S> {
     // But it would be ok to overlap our new request with receiving
     // of a previous response.
     //
-    // TODO: The RoT must be able to observe SP resets. During the
+    // The RoT must be able to observe SP resets. During the
```

Comment:
I think this is still a TODO, right?

Reply:
Yes. It should be an issue if it's not already.